src/dodal/beamlines/p99.py

```diff
@@ -53,9 +62,44 @@ def sample_lab_xyz_stage(
     wait_for_connection: bool = True, fake_with_ophyd_sim: bool = False
 ) -> XYZPositioner:
     return device_instantiation(
-        FilterMotor,
+        XYZPositioner,
         prefix="-MO-STAGE-02:LAB:",
         name="sample_lab_xyz_stage",
         wait=wait_for_connection,
         fake=fake_with_ophyd_sim,
     )
+
+
+andor_data_path = StaticPathProvider(
+    filename_provider=AutoIncrementFilenameProvider(base_filename="andor2"),
+    directory_path=Path("/dls/p99/data/2024/cm37284-2/processing/writenData"),
+)
+
+
+def andor2_det(
+    wait_for_connection: bool = True, fake_with_ophyd_sim: bool = False
+) -> Andor2:
+    return device_instantiation(
+        Andor2,
+        prefix="-EA-DET-03:",
+        name="andor2_det",
+        path_provider=andor_data_path,
+        wait=wait_for_connection,
+        fake=fake_with_ophyd_sim,
+    )
+
+
+def andor2_point(
```

Comment:
Could: As someone not too familiar with p99 it's not clear to me what the difference is between `andor2_det` and `andor2_point` and when I should use one over the other. I think putting some docstrings on them would be good